Cooper/DAO/GameDAO.cs

```diff
@@ -0,0 +1,190 @@
+    {
+
+        private CRUD crud;
+        Logger logger;
+
+        private string table;
+        private string idColumn;
+        private List<string> attributes;
+
+
+        public GameDAO()
+        {
+            crud = new CRUD();
+            logger = LogManager.GetLogger("CooperLoger");
+
+            table = "GAMES";
+            idColumn = "ID";
+            attributes = new List<string>()
+            {
+                "ID", "NAME", "DESCRIPTION", "GENRE", "LINK", "LOGOURL",
+                "COVERURL", "ISVERIFIED"
+            };
+        }
+
+        public GameDb Get(long id)
+        {
+            EntityORM entity = crud.Read(id, table, idColumn, attributes);            
+
+            GameDb game = MapEntity(entity);
+
+            return game;
+        }
+
+        public IEnumerable<GameDb> GetAll()
+        {
+            List<GameDb> games = new List<GameDb>();
+
+            List<EntityORM> entities = (List<EntityORM>)crud.ReadAll(table, attributes);
+
+            foreach (EntityORM entity in entities)              // Mapping entities to games
+            {
+                GameDb game = MapEntity(entity);
+                games.Add(game);
+            }
+
+            return games;
+        }
+
+        ///<summary>
+        ///Maps properties from EntityORM object to GameDb object
+        ///</summary>
+        private GameDb MapEntity(EntityORM entity)
+        {
+            GameDb game = new GameDb();
+
+            foreach (AttributeValue aV in entity.attributeValue)
+            {
+                switch(aV.Attribute)
+                {
+                    case "ID":
+                        game.id = Convert.ToInt64(aV.Value);
+                        break;
+                    case "NAME":
+                        game.Name = aV.Value.ToString();
+                        break;
+                    case "DESCRIPTION":
+                        game.Description = aV.Value.ToString();
+                        break;
+                    case "GENRE":
+                        game.Genre = aV.Value.ToString();
+                        break;
+                    case "LINK":
+                        game.Link = aV.Value.ToString();
+                        break;
+                    case "LOGOURL":
+                        game.LogoURL = aV.Value.ToString();
+                        break;
+                    case "COVERURL":
+                        game.CoverURL = aV.Value.ToString();
+                        break;
+                    case "ISVERIFIED":
+                        game.isVerified = ((char)aV.Value == 'y') ? true : false;
+                        break;
+                    default:
+                        break;
+                }
+            }
+
+            return game;
+        }
+        
+        ///<summary>
+        ///Maps properties from GameDb object to EntityORM object
+        ///</summary>
+        private EntityORM MapEntity(GameDb game)
```

Comment:
SOLID, letter S states for single responsibility, so one class has a single job. Move mappers to another class Mapper which will have all methods like 
public EntityOrm ToOrm(GameDb game), public EntityOrm ToOrm(ChatDb chat), ...
and reverse methods:
public GameDb ToGame(EntityOrm entity), public ChatDb ToChat(EntityOrm chat), ....